data/default/health/field-app-configuration.json

```diff
@@ -32,6 +32,33 @@
           "name": "OTHER"
         }
       ],
+
+      "HOUSEHOLD_DELETION_REASON_OPTIONS":[
+
```

Comment:
please always format code before committing

Reply:
@naveen-egov please try to make sure the formatting of the file is consistent after you have made changes. Note this commit - https://github.com/egovernments/health-campaign-mdms/pull/67/commits/50004085b93d6266251eb986b2aebe0a7324e4cb